<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TaffesirListManagerWidget</class>
 <widget class="QWidget" name="TaffesirListManagerWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>486</width>
    <height>347</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>التفاسير</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::RightToLeft</enum>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTreeView" name="treeView">
     <property name="rootIsDecorated">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QToolButton" name="toolMoveUp">
       <property name="minimumSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../share/moltaqa-lib/qt_rc.qrc">
         <normaloff>:/images/go-up2.png</normaloff>:/images/go-up2.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>22</width>
         <height>22</height>
        </size>
       </property>
       <property name="autoRepeat">
        <bool>true</bool>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toolMoveDown">
       <property name="minimumSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../share/moltaqa-lib/qt_rc.qrc">
         <normaloff>:/images/go-down2.png</normaloff>:/images/go-down2.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>22</width>
         <height>22</height>
        </size>
       </property>
       <property name="autoRepeat">
        <bool>true</bool>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../share/moltaqa-lib/qt_rc.qrc"/>
 </resources>
 <connections/>
</ui>
